styles/native/js/core/widgets/linechart.js

```diff
@@ -0,0 +1,94 @@
+import { border, brand, font } from "../variables";
+/*
+
+DISCLAIMER:
+Do not change this file because it is core styling.
+Customizing core files will make updating Atlas much more difficult in the future.
+To customize any core styling, copy the part you want to customize to styles/native/app/ so the core styling is overwritten.
+
+==========================================================================
+    Line Chart
+
+    Default Class For Mendix Line Chart Widget
+========================================================================== */
+// eslint-disable-next-line @typescript-eslint/camelcase
+export const com_mendix_widget_native_linechart_LineChart = {
+    container: {
+    // All ViewStyle properties are allowed
+    },
+    errorMessage: {
+        // All TextStyle properties are allowed
+        fontFamily: font.family,
+        fontSize: font.sizeSmall,
+        fontWeight: font.weightNormal
+    },
+    chart: {
+    // All ViewStyle properties are allowed
+    },
+    grid: {
+        lineColor: border.color,

```

Comment:
This object is missing a comment

Reply:
Added a comment with allowed properties.